spec/fixtures/files/service_history/cas_activity_methods/source/Enrollment.csv

```diff
@@ -1,3 +1,4 @@
 EnrollmentID,PersonalID,ProjectID,EntryDate,HouseholdID,RelationshipToHoH,LivingSituation,LengthOfStay,LOSUnderThreshold,PreviousStreetESSH,DateToStreetESSH,TimesHomelessPastThreeYears,MonthsHomelessPastThreeYears,DisablingCondition,DateOfEngagement,MoveInDate,DateOfPATHStatus,ClientEnrolledInPATH,ReasonNotEnrolled,WorstHousingSituation,PercentAMI,LastPermanentStreet,LastPermanentCity,LastPermanentState,LastPermanentZIP,AddressDataQuality,DateOfBCPStatus,EligibleForRHY,ReasonNoServices,RunawayYouth,SexualOrientation,SexualOrientationOther,FormerWardChildWelfare,ChildWelfareYears,ChildWelfareMonths,FormerWardJuvenileJustice,JuvenileJusticeYears,JuvenileJusticeMonths,UnemploymentFam,MentalHealthIssuesFam,PhysicalDisabilityFam,AlcoholDrugAbuseFam,InsufficientIncome,IncarceratedParent,ReferralSource,CountOutreachReferralApproaches,UrgentReferral,TimeToHousingLoss,ZeroIncome,AnnualPercentAMI,FinancialChange,HouseholdChange,EvictionHistory,SubsidyAtRisk,LiteralHomelessHistory,DisabledHoH,CriminalRecord,SexOffender,DependentUnder6,SingleParent,HH5Plus,IraqAfghanistan,FemVet,HPScreeningScore,ThresholdScore,VAMCStation,DateCreated,DateUpdated,UserID,DateDeleted,ExportID
 "1-1","1-1","1-1","2015-11-07","","","","","","","","","","99","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",,"2015-09-10 13:37:31","2015-11-11 00:17:33","catalina","","1-1"
 "1-3","1-2","1-2","2016-01-01","","","","","","","","","","99","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",,"2016-09-10 13:37:31","2016-11-11 00:17:33","catalina","","1-1"
+1-4,1-1,1-3,2017-01-01,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-01-01 00:00:00,2017-01-01 00:00:00,catalina,,1-1
```

Comment:
For good measure, can you add another enrollment in CE that **doesn't** have an assessment?